webapp/channels/src/tests/react_testing_utils.tsx

```diff
@@ -39,7 +40,7 @@ export const renderWithContext = (
         useMockedStore: partialOptions?.useMockedStore ?? false,
     };
 
-    const testStore = configureOrMockStore(initialState, options.useMockedStore);
+    const testStore = configureOrMockStore(initialState, options.useMockedStore, partialOptions?.pluginReducers);
```

Comment:
Can you pass `pluginReducers` into the other uses of `configureOrMockStore` below as well?